modules/rosa-sdpolicy-platform.adoc

```diff
@@ -26,7 +26,7 @@ endif::rosa-with-hcp[]
 
 [IMPORTANT]
 ====
-It is critical that customers have a backup plan for their applications and application data.
+There is no Red Hat-provided backup method available for ROSA clusters with STS, which is the default. It is critical that customers have a backup plan for their applications and application data. 
```

Comment:
I don't feel great about this hyphenated Red Hat-provided. I suggest the following rework:
Red Hat does not provide a backup method for ROSA clusters with STS, which is the default.